### ISSUE TITLE
Use a resnet model for swarm learning

### ISSUE THREAD

**Issue by oliversaldanha25** [HewlettPackard/swarm-learning#47]
I have asked this question before. I just want to know how can we use a transfer learning model in the swarm learning pipeline. It is giving the following error.
![transfer_learnning_error](https://user-images.githubusercontent.com/77096343/130782208-fe603c17-356e-4088-b692-e29d66b1f9b9.png)
Do i have to recreate the model architecture?

**Comment by RadhakrishnaJ** [comment]
> > > I am able to use alexnet with pretrained weights.
> > > But i am not able to use other complex models such as resnet, shufflenet, efficientnet?
> > > This is a very useful feature for image classification problems.
> > > Is there any possibility that we can use there models?
> > > What can i do to try out something which can help me use resnet for my application.
> > > I would really appreciate any help in this topic.
> > > Looking forward for the reply?
> > 
> > 
> > Hello,
> > From your reply I understood that you could able to run Swarm with Alexnet pre-trained model, this is a good sign. I hope you are using AlexNet composed of 5 convolutional layers followed by 3 fully connected layers.
> > When it comes to RESNET, many variants are available. Not sure which RESNET architecture you are using. I feel your RESNET is comparably heavy with respect to number of weight parameters when compared to Alexnet. At this moment without looking at code, there could be a chance your resources (processing and memory) may not be sufficient for running RESNET model
> > Could you confirm whether your machine could able to run your RESNET ML model without Swarm callback? This helps to determine your resources capacity. You need to try this in your standalone ML platform.
> 
> Thanks for the reply.
> I am using resnet 18. I have tried using the same model stand-alone and it works fine.

Could you please confirm whether you could able to get RESNET ML working? 
If not, could you please provide more details as per this template to assist further on this issue. 
[Swarm_Learning_issue_47_reporting_template.docx](https://github.com/HewlettPackard/swarm-learning/files/7089034/Swarm_Learning_issue_47_reporting_template.docx)

**Comment by RadhakrishnaJ** [comment]
> > > Hello,
> > > As a community user, if you would like to contribute you can share your AlexNet based transfer learning model that helps other users as well.
> > > If you feel your dataset is confident, the same can be applied with the public dataset to share your model. We need your consent so that we can provide you special permissions on this repository.
> > 
> > 
> > This would be a great opportunity to improve my workflow and also contribute to the swarm learning framework.
> > Can we have a small discussion via email so that i have certain things to clarify and explain how i plan to make this transfer learning model with the public dataset available for everyone to use?
> > There is a paper which we are planning to publish based on our findings on swarm learning and would like HPE to be a part in the collaboration since i have received lot of support from @RadhakrishnaJ and others.
> > My email id is [osaldanha@ukaachen.de](mailto:osaldanha@ukaachen.de)
> > looking forward for a email and a small discussion time.
> 
> Thanks for your response. We will connect with you through mail. We have a long weekend, by Tuesday we will get back to you with details.

Sorry for the delayed response. 

- To contribute your code snippet to Swarm Learning, you can work out your ML code with public dataset and add your code in new folder inside examples, once the pull request is raised after 2 members approval from our team it will be published. Let us know if you still have questions. 
  
-  With respect to discussion, if it is a general discussion item please see if you can use discussion forum so that discussion will help rest of the community users. 
-  If you wanted to have a discussion on your paper or any specific support, please let us know we will get back to you on other communication channels.

**Comment by oliversaldanha25** [comment]
Thanks for the reply

- There are some specific questions that need to be answered so I think a discussion forum would not be the right place to start.
- After the discussion, I can make an example folder and upload it on Github and give a pull request it is not a problem.
Looking forward to your reply
